packages/just-stack-react/package.json

```diff
@@ -15,6 +15,7 @@
     "autoprefixer": "^9.4.5",
     "beachball": "^1.11.5",
     "css-loader": "^2.1.0",
+    "cssnano": "^4.1.10",
```

Comment:
why are you adding the deps here not in `just-scripts/package.json` ?

Reply:
Historically I think @kenotron has wanted to keep deps of `just-scripts` itself pretty minimal because not everyone will want all the optional deps. Unfortunately this means the extra `just-scripts` functionality relies on entirely implicit dependencies, which is not ideal, but last we discussed it there wasn't a better way (at least that we could both agree on).

Reply:
@ecraig12345 is right - that's the philosophy for just-scripts. The stack is the place to add this, so this change is right.

Reply:
Alright; i've backed out the package.json addition and yarn lock updates.